src/styles/index.scss

```
@@ -1,3 +1,120 @@
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-secondary-color);
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: var(--text-accent-color);
+  }
+  &__code {
+    margin-bottom: 16px;
+  }
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    margin-top: 16px;
+    box-sizing: border-box;
+    height: 40px;
+    width: 166px;
+    border: 1px solid var(--main-color);
+    border-radius: 5px;
+
+    font-size: 14px;
+    line-height: 16px;
+    color: white;
+    font-weight: 700;
+    text-decoration: none;
+
+    background-color: var(--main-color);
+  }
+}
+
+.card__review-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
```

Use scss nesting, fix it everywhere
```suggestion
  }
  
  &__review-container {
    // write styles here
  }
}

```